save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -481,13 +455,3 @@ val cosvFileManagerComponent: FC<Props> = FC {
         }
     }
 }
-
-private fun RawCosvFileDto.notSelectableReason() = when {
-    status == RawCosvFileStatus.PROCESSED -> "Already processed"
-    status == RawCosvFileStatus.IN_PROGRESS -> "In progress, please wait"
-    isDuplicate() -> "Duplicate, the vulnerability with such ID already uploaded"
-    isUploadedZipArchive() -> "It's a zip archive, please unzip to get JSON files"
```

Comment:
what will be with zip arhives now? if we press the button `submit` before `unzip`?

Reply:
They will not be affected since `submit` only applies to JSON files. They will remain in uploaded list.